app/partials/header.html

```diff
@@ -1,8 +1,31 @@
 <div class="header">
-  <ul class="nav nav-pills pull-right">
-    <li class="active"><a href="#">Home</a></li>
-    <li><a href="#">About</a></li>
-    <li><a href="#">Contact</a></li>
-  </ul>
-  <h3 class="text-muted">PageTraining</h3>
+  <div class="row">
+  <div class="col-lg-2 companyname">
+  </div>
+  <div class="col-lg-3 col-lg-offset-2">
+    <form>
```

Comment:
Take a look at boostrap classes for <form>s, this way you can style this very quickly